src/minigames/assets.ts

```diff
@@ -43,6 +43,7 @@ export enum MINIGAME_ASSET_ALIASES {
   PROTEIN = "protein",
   STRAWBERRIES = "strawberries",
   YOGHURT = "yoghurt",
+<<<<<<< Updated upstream
```

Comment:
These are merge conflict markers, there's a few in this file. I think we can just remove all the <<<<<<<<<, ========= and >>>>>>> lines (use both sets of changes, everywhere in this file)